appinventor/appengine/src/com/google/appinventor/client/wizards/youngandroid/NewYoungAndroidProjectWizard.java

```diff
@@ -56,7 +56,7 @@ public NewYoungAndroidProjectWizard(final ProjectToolbar toolbar) {
       @Override
       public boolean validate(String value) {
         errorMessage = TextValidators.getErrorMessage(value);
-        if (errorMessage.length()>0){
+        if (errorMessage.length()>0){ 
```

Comment:
Not sure why this is changed in this commit.

Reply:
Please Ignore this, I forgot to form a branch When solving this issue.

Reply:
Fixed